### DISCUSSION TITLE
捐赠

### DISCUSSION THREAD

**Discussion by hmgdd101** [cotaku/Bibi-Release#23] [General]
老板可以公布个捐赠渠道吗，我之前找了好久没找到，真实运气好才发现有人说你没放弃，终于找来了。
如果可以的话放一个捐赠渠道，我愿意请你吃顿大餐，我相信也会有其他人愿意给你付款。
只要你别走

**Comment by cotaku** [comment]
感谢心意，不过现在主要是业余有空的时候搞一搞，收赞助的话性质就不一样了，会让我鸭梨山大的 ><
等以后万一哪天被资本家优(cai)化(yuan)掉了的时候可以考虑一下（逃
